package.json

```diff
@@ -25,7 +25,7 @@
   "author": "Gustavo Santos (JBay Solutions) <gustavo.santos@jbaysolutions.com> (http://www.jbaysolutions.com)",
   "dependencies": {
     "element-resize-detector": "^1.1.10",
-    "interact.js": "^1.2.8"
+    "interact.js": "git://github.com/taye/interact.js.git#master"
```

Comment:
You are right, but they are a bit slow on developing, that feature was already ready months ago and I waited for them to publish on npm as I needed it for my project. Unfortunately, it has not been published yet (they have released version 1.3.0alpha which is in the unstable branch and is not available through package.json). So I decided to implement it through the master branch. Feel free to not commit my changes, as soon as the new version of the interactjs library will be published you just have to edit your package.json